autopush/utils.py

```diff
@@ -142,7 +142,11 @@ def extract_jwt(token, crypto_key):
     # on the JWT object. Vapid is a bit more creative in how it
     # stores data into a JWT and breaks expectations. We would have to
     # turn off most of the validation in order for it to be useful.
-    return json.loads(jws.verify(token, vk, algorithms=["ES256"]))
+    return jwt.decode(token, vk, options=dict(
+        verify_aud=False,
+        verify_sub=False,
+        verify_exp=False,
```

Comment:
Put simply, we are validating the contents, just not via JWT. We have additional code checks which return descriptive error messages for the important bits. In addition, our use of sub isn't quite to spec with what jwt expects.